app/src/main/java/com/kickstarter/ApplicationModule.java

```diff
@@ -133,16 +134,15 @@ static Environment provideEnvironment(final @NonNull @ActivitySamplePreference I
     final @NonNull InternalToolsType internalToolsType,
     final @NonNull KSCurrency ksCurrency,
     final @NonNull KSString ksString,
-    final @NonNull @LakeTracker Koala lake,
+    final @NonNull @LakeTracker AnalyticEvents analytics,
```

Comment:
What is the `@LakeTracker` annotation doing here? Do we need to keep this if the AnalyticsEvents class is not supposed to be specific to one event tracker?